ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -1,8 +1,140 @@
 import QtQuick 2.3
+import "../../../imports"
 
 Item {
-    Text {
-        id: name3
-        text: "HISTORY"
+    Component {
+      id: transactionListItem
+
+      Item {
+        anchors.right: parent.right
+        anchors.rightMargin: 0
+        anchors.left: parent.left
+        anchors.leftMargin: 0
```

Comment:
Just an FYI, but you can remove the margin lines if they are 0 since it's the default.